Numbers/ComplexNumber.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Numbers;
+/// <summary>
+/// Класс комплексных чисел
+/// </summary>
+public class ComplexNumber
+{
+    private float _Real;
+    private float _Imaginary;
```

Comment:
```suggestion
    private readonly double _Real;
    private readonly double _Imaginary;
```

1. `float` используется в очень узких областях - где экономят память, либо быстродействие.
2. Если структура "только для чтения", то здесь поля надо сделать "только для чтения".